src/js/views/Reports/NamespaceKPIs.jsx

```diff
@@ -57,20 +80,41 @@ function NamespaceKPIs() {
             data: result,
             fetched: true,
             lookup: lookup,
-            errorMessage: null
+            errorMessage: null,
+            sort: state.sort
           })
```

Comment:
There is still an issue here.  If we add a new member to the state that we want to keep around, then we will clobber it without a spread.
```javascript
setState({ ...state, {data: result, fetched:true, errorMessage:null}})
```